types/env/const.go

```diff
@@ -1,5 +1,9 @@
 package env
 
 const (
-	DatabaseURI = "JUNO_DATABASE_URL"
+	DatabaseURI           = "JUNO_DATABASE_URL"
+	DatabaseSSLModeEnable = "false"
+	DatabaseSSLRootCert   = ""
+	DatabaseSSLCert       = ""
+	DatabaseSSLKey        = ""
```

Comment:
These should be env variables names, not values
```suggestion
	DatabaseSSLModeEnable = "JUNO_DATABASE_SSL_MODE_ENABLED"
	DatabaseSSLRootCert   = "JUNO_DATABASE_SSL_ROOT_CERT"
	DatabaseSSLCert       = "JUNO_DATABASE_SSL_CERT"
	DatabaseSSLKey        = "JUNO_DATABASE_SSL_KEY"
```